### DISCUSSION TITLE
Can't upload files to Chamilo 1.11

### DISCUSSION THREAD

**Discussion by Keerthi1973** [chamilo/chamilo-lms#5174] [General]
Dear Expertise,
I already installed Chamilo in XAMPP in localhost and it was operate very correctly. But when I migrate it to paid hosting place I can't upload any file (even documents to course, user photo ect. ) to Chamilo. I use Chamilo 1.11.26.
Files are upload until 98% and then "The file upload failed" message popup and stop the operation. Also when I check the relavant folder at the server, the files can be found exist, but those are not appear as expected at the Cahmilo portal.
Please help.. Thank you.. 
Keerthi

**Comment by a-mian-lateral** [comment]
Hosting providers usually keep track of error logs, check for any related log and post it here, that will help to find the cause.

**Reply by Keerthi1973** [reply]
Thanks you, I will check and let you know..
